docs/examples/codecs/handling-documents/using-codec.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+use MongoDB\Client;
+
+$client = new Client();
+$collection = $client->selectCollection('test', 'person', [
+    'codec' => new PersonCodec(),
+]);
+
+$person = new Person('Jane Doe');
+$collection->insertOne($person);
+
+$person = $collection->findOne();
```

Comment:
Would using a different variable name make more sense here? I think I've tried to do that in other example code, but it's by no means a rule.

I'll defer you to ignore/resolve.